mage_ai/services/spark/spark.py

```diff
@@ -2,6 +2,33 @@
 import os
 
 
+def get_list_of_file_names(list_of_jars):
+    if list_of_jars is None:
+        return None
+
+    return [os.path.basename(jar) for jar in list_of_jars]
+
+
+def contains_same_jars(list_of_jars_1, list_of_jars_2):
```

Comment:
nit: add type
add method docstring

Reply:
could you also add unit test to this method?

Reply:
Made the suggested change.